R/test_protocol_wrapper.R

```diff
@@ -0,0 +1,127 @@
+# This file is a wrapper for the functions in test_general_format.R
+# It allows us to run all tests for a given pipeline in one go
+
+#' Test protocol compliance
+#'
+#' Runs all protocol validation tests from test_general_format.R
+#'
+#' @param pipeline_output A list of data frames returned from format_X,
+#' where X is the pipeline code.
+#' @param protocol_version The protocol version of the SPI-Birds standard
+#' data being used. Either "1.0.0", "1.1.0", or "2.0.0".
+#' @param ID_format Regular expression pattern for validating ID formats.
+#' Default is FALSE, which skips this test.
+#'
+#' @return Invisible TRUE if all tests pass, errors otherwise.
+#' @export
+
+test_protocol_compliance <- function(pipeline_output,
+                                     protocol_version,
```

Comment:
As the protocol version is part of pipeline output (as the fifth item in the list), does it need to be a separate argument here?